expected/arraybuffer.out

```diff
@@ -0,0 +1,13 @@
+CREATE FUNCTION crash_test() RETURNS boolean
+LANGUAGE plv8 IMMUTABLE STRICT
+AS $$
+  var a = new ArrayBuffer(8);
+  return true;
+$$;
+SELECT crash_test();
+WARNING:  Allocating: 8
```

Comment:
no, it's an artifact of me not getting the correct expected file from my vm - fixing now.